tests/test_reports.py

```diff
@@ -151,7 +151,7 @@ def test_reports_sort_default(self):
             datetime.datetime(2015, 7, 5),
         ]
         dates_formatted = [isoformat(each) for each in dates]
-        reports = [
+        [
```

Comment:
This is super minor, since it works fine, but few lines up, in `test_reports_sort`, those reports are assigned to `reports`. It might be nice to keep that consistent.